### PR TITLE
express-fileupload busboy options

### PR DESCRIPTION
**Pull request by markxoe**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/richardgirges/express-fileupload/pull/275
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

The optional busboy options as described [here](https://github.com/richardgirges/express-fileupload#using-busboy-options) were not included in this types module

### CODE REVIEW

File: types/express-fileupload/index.d.ts [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 // Project: https://github.com/richardgirges/express-fileupload#readme
 // Definitions by: Gintautas Miselis <https://github.com/Naktibalda>
 //                 Sefa Ilkimen <https://github.com/silkimen>
 //                 Tomas Vosicky <https://github.com/vosatom>
 //                 Piotr Błażejewicz <https://github.com/peterblazejewicz>
+//                 Mark Oude Elberink <https://github.com/markxoe>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 import express = require('express');
+import busboy = require('busboy');
```
Comment (peterblazejewicz) [inline types/express-fileupload/index.d.ts:11]: please use `import type { BusboyConfig } `
DT is now on 3.8, so one can point actual intent (import type, not actual symbols),

thx!

Comment (markxoe) [reply types/express-fileupload/index.d.ts:11]: Is `import type { BusboyConfig } from 'busboy';`
or `import type busboy = require('busboy');` better in this case?

Comment (markxoe) [reply types/express-fileupload/index.d.ts:11]: Guess I'll go with
```
import type { RequestHandler } from 'express';
import type { BusboyConfig } from 'busboy';
```
just to enforce code style

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@markxoe thx!

**Comment by markxoe** [comment]
Ready to merge

**Pull request merged**